spec.html

```diff
@@ -50961,7 +50961,7 @@ <h1>Changes to FunctionDeclarationInstantiation</h1>
         <p>During FunctionDeclarationInstantiation the following steps are performed in place of step <emu-xref href="#step-functiondeclarationinstantiation-web-compat-insertion-point"></emu-xref>:</p>
         <emu-alg replaces-step="step-functiondeclarationinstantiation-web-compat-insertion-point">
           1. If _strict_ is *false*, then
-            1. For each |FunctionDeclaration| _f_ that is directly contained in the |StatementList| of a |Block|, |CaseClause|, or |DefaultClause|, do
+            1. For each |FunctionDeclaration| _f_ that is directly contained in the |StatementList| of a |Block|, |CaseClause|, or |DefaultClause| Contained within _code_, do
```

Comment:
In the editor call, we decided that we would prefer the form `... any |Block|, |CaseClause|, or |DefaultClause| _x_ such that _code_ Contains _x_ is *true*` that [you mentioned](https://github.com/tc39/ecma262/issues/2663#issuecomment-1038441337). We will explore dropping the `is *true*` from here and in other places in #3370.

We should also update the two existing uses of `Contained within` in B.3.2.2 and B.3.2.3 to match this phrasing.

Reply:
Added a commit to accomplish this.